README.md

```diff
@@ -1,2 +1,21 @@
-# personal-portfolio
-Portfolio Website Project
+# Personal Portfolio Website
+Set up and Mobile Version Skeleton.
+
+# Built With
+- HTML
+- CSS
+- GitHub
+- Linters
+
+# To Run
+To run this project, you need to have a web browser such as Google Chrome, Safari, etc. And you need to clone the repository or download the ZIP files.
```

Comment:
- [ ][OPTIONAL] Kindly include commands for executing the stated steps for example cloning the repo to best help all kinds of visitors.